trio/_ssl.py

```diff
@@ -741,11 +760,11 @@ async def unwrap(self):
             await self._handshook.ensure(checkpoint=False)
             await self._retry(self._ssl_object.unwrap)
             transport_stream = self.transport_stream
-            self.transport_stream = None
             self._state = _State.CLOSED
+            self.transport_stream = None  # type: ignore[assignment]  # Messy, but State closed, should not use, should be fine...
```

Comment:
Make the comment a bit more verbose to explain what's going on here

Reply:
Changed in edbf981

Reply:
?
That's a 3 day old commit and doesn't look to contain any changes to the comment